### PR TITLE
Switch to Yocto-3.15

### PR DESCRIPTION
**Pull request by iartemenko**

### CODE REVIEW

**Comment by otyshchenko1** [comment]
meta-xt-images should contain a patch for setting RCAR_BL33_EXECUTION_EL...
Plaese see 0001-Add-option-for-arm-trusted-firmware.patch for details.

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-bsp/arm-trusted-firmware/arm-trusted-firmware_git.bbappend [hunk 1]
```diff
@@ -11,4 +11,3 @@ do_deploy_append () {
     install -m 0644 ${S}/tools/dummy_create/cert_header_sa6_emmc.bin ${DEPLOYDIR}/cert_header_sa6_emmc.bin
     install -m 0644 ${S}/tools/dummy_create/cert_header_sa6_emmc.srec ${DEPLOYDIR}/cert_header_sa6_emmc.srec
 }
-
```
Comment (otyshchenko1) [inline recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-bsp/arm-trusted-firmware/arm-trusted-firmware_git.bbappend]: please remove this change

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-bsp/arm-trusted-firmware/files/0001-rcar-Use-UART-instead-of-Secure-DRAM-area-for-loggin.patch [hunk 2]
```diff
@@ -1,7 +1,7 @@
-From 0bdaa54e0f72443930b066251ff276d282c7a2b4 Mon Sep 17 00:00:00 2001
+From b199ba985199f0dd56553d1e3fd2a1570317a57a Mon Sep 17 00:00:00 2001
```
Comment (otyshchenko1) [inline recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-bsp/arm-trusted-firmware/files/0001-rcar-Use-UART-instead-of-Secure-DRAM-area-for-loggin.patch]: Please split this commit into two patches in order to separate changes for fixing ARM TF build and (0001-rcar-gen3-arm-trusted-firmware-Allow-to-add-more-bui.patch) and aligning existing commit with new ARM TF version (0001-rcar-Use-UART-instead-of-Secure-DRAM-area-for-loggin.patch)

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-bsp/u-boot/u-boot_2018.09.bbappend [hunk 3]
```diff
@@ -0,0 +1,5 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+
+##SRC_URI += "
+##    file://0003-salvator-x-Don-t-disable-RAVB-module.patch
+##"
```
Comment (otyshchenko1) [inline recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-bsp/u-boot/u-boot_2018.09.bbappend]: Please drop this patch

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-bsp/arm-trusted-firmware/files/0001-rcar-Use-UART-instead-of-Secure-DRAM-area-for-loggin.patch (hunk 2 above)
Comment (iartemenko) [reply recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-bsp/arm-trusted-firmware/files/0001-rcar-Use-UART-instead-of-Secure-DRAM-area-for-loggin.patch]: Done

**Comment by otyshchenko1** [comment]
I will close this PR and create a new one instead...

**Pull request closed without merging**